app/models/publisher.rb

```diff
@@ -103,6 +103,32 @@ class Publisher < ApplicationRecord
 
   store_accessor :feature_flags, VALID_FEATURE_FLAGS
 
+  def self.uphold_creators
+    joins(:uphold_connection).
+      where('uphold_connections.country IS NULL').
+      where(selected_wallet_provider_type: 'UpholdConnection').
+      or(
+        joins(:uphold_connection).
+        where.not("uphold_connections.country ILIKE '#{UpholdConnection::JAPAN}'").
+        where(selected_wallet_provider_type: 'UpholdConnection')
+      )
+  end
+
+  def self.gemini_creators
+    joins(:gemini_connection).
+      where('gemini_connections.country IS NULL').
+      where(selected_wallet_provider_type: 'GeminiConnection').
+      or(
+        joins(:gemini_connection).
+        where.not("gemini_connections.country ILIKE '#{GeminiConnection::JAPAN}'").
+        where(selected_wallet_provider_type: 'GeminiConnection')
+      )
+  end
+
+  def self.bitflyer_creators
+    where(selected_wallet_provider_type: 'BitflyerConnection')
+  end
```

Comment:
Nit: I think we should use constants. This particular column has given me issues.

Reply:
Good idea, I'll update!